modules/Microsoft.Sql/servers/.test/common/deploy.test.bicep

```diff
@@ -116,6 +116,10 @@ module testDeployment '../../deploy.bicep' = {
         diagnosticEventHubAuthorizationRuleId: diagnosticDependencies.outputs.eventHubAuthorizationRuleId
         diagnosticEventHubName: diagnosticDependencies.outputs.eventHubNamespaceEventHubName
         elasticPoolId: '${resourceGroup.id}/providers/Microsoft.Sql/servers/<<namePrefix>>-${serviceShort}/elasticPools/<<namePrefix>>-${serviceShort}-ep-001'
+        encryptionProtectorObj: {
```

Comment:
Hey @jhueppauff thanks for adding everything commented. Looks good from my end.

Just one last question. Usually we would ask the contributor to provide a pipeline badge from their fork to show that the new code works in the CI pipeline. Do you happen to have a corresponding pipeline set up?

If not and you don't want to, I could offer to merge the PR into a branch and test it from my end.

Reply:
I seem to have some issues with the pipeline.
https://github.com/jhueppauff/ResourceModules/actions/runs/4479941877/jobs/7874817993 
But the errors look a bit unrelated to the changes

Reply:
It is unrelated. The `namePrefix` you chose when setting up your CI just kicks the test storage accounts name above the allowed length of 24 characters. Please remove one character and try again.

Reply:
Ok, got it now working expect the publishing part:
https://github.com/jhueppauff/ResourceModules/actions/runs/4479941877

Is there an option to override the registry name? Updating the yml seems to be wrong for the branch.

Reply:
It's unfortunately the nature of git in this case - we cannot exactly 'gitignore' specific lines in a file.

However, we're currently looking into incorporating variable groups more and may implement an option to use those instead - simply to enable people to avoid exactly this hassle. It's not exactly a 'nice' solution, but I guess the best option we've got.

Reply:
Ok, I updated it :)
Pipeline is green now